<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbZaloguj">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zaloguj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbZarejestruj">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zarejestruj </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbWyloguj">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyloguj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbWyswietl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl filmy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDodaj">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj filmy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbSortowanie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Sortowanie</string>
    </property>
    <widget class="QRadioButton" name="rbAlfabet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Alfabetycznie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbOcena">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ocena</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbLwys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Liczba wys.</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbWypozycz">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypożycz film</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Oddaj">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Oddaj film</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Wyszukaj">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyszukaj film</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Ocen">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Oceń film</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
